src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -118,6 +126,9 @@ public SwerveDriveKinematics getKinematics() {
     }
 
     public Rotation2d getGyroscopeRotation() {
+        // double rot = Math.abs(pigeon2.getYaw()) % 360 * ( (pigeon2.getYaw() < 0) ? -1 : 1);
+        // if (rot<0) rot += 360;
+        // return Rotation2d.fromDegrees(rot);
```

Comment:
Is any of this commented out section still necessary?

Reply:
I don't believe so for following trajectories. This could be useful for an absolute angle measurement, though